docs/docs/llm-evaluate/index.mdx

```diff
@@ -0,0 +1,739 @@
+            ],
+        }
+    )
+
+
+    def openai_qa(inputs: pd.DataFrame) -> List[str]:
+        predictions = []
+        system_prompt = "Please answer the following question in formal language."
+
+        for _, row in inputs.iterrows():
+            completion = openai.chat.completions.create(
+                model="gpt-4o-mini",
+                messages=[
+                    {"role": "system", "content": system_prompt},
+                    {"role": "user", "content": row["inputs"]},
+                ],
+            )
+            predictions.append(completion.choices[0].message.content)
+
+        return predictions
+
+
+    with mlflow.start_run():
+        results = mlflow.evaluate(
+            model=openai_qa,
+            data=eval_data,
+            targets="ground_truth",
+            model_type="question-answering",
+        )
+
+    print(results.metrics)
+```
+
+```python
+    :caption: Output
+
+    {
+        "flesch_kincaid_grade_level/v1/mean": 14.75,
+        "flesch_kincaid_grade_level/v1/variance": 0.5625,
+        "flesch_kincaid_grade_level/v1/p90": 15.35,
+        "ari_grade_level/v1/mean": 18.15,
+        "ari_grade_level/v1/variance": 0.5625,
+        "ari_grade_level/v1/p90": 18.75,
+        "exact_match/v1": 0.0,
+    }
+```
+
+### Evaluating with an MLflow Deployments Endpoint
+
+For MLflow >= 2.11.0, [mlflow.evaluate()](<../python_api/mlflow.mdx#mlflow.evaluate()>) supports evaluating a model endpoint by directly passing the MLflow Deployments endpoint URI to the `model` argument.
+This is particularly useful when you want to evaluate a deployed model hosted by a local [MLflow AI Gateway](../deployments/index.html), [Databricks Foundation Models API](https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html), and [External Models in Databricks Model Serving](https://docs.databricks.com/en/generative-ai/external-models/index.html), without implementing custom prediction logic to wrap it as an MLflow model or a python function.
+
+Please don't forget to set the target deployment client by using [mlflow.deployments.set_deployments_target()](../python_api/mlflow.mdx#mlflow.deployments.set_deployments_target) before calling [mlflow.evaluate()](<../python_api/mlflow.mdx#mlflow.evaluate()>) with the endpoint URI, as shown in the example below. Otherwise, you will see an error message like `MlflowException: No deployments target has been set...`.
+
+<Hint />
+
+    When you want to use an endpoint **not** hosted by an MLflow AI Gateway or Databricks, you can create a custom Python function following the [Evaluating with a Custom Function](../llms/llm-evaluate/index.html#llm-eval-custom-function)` guide and use it as the ``model`` argument.
```

Comment:
instead of a custom component like this, could we use the [admonitions syntax](https://docusaurus.io/docs/next/markdown-features/admonitions)? 

```suggestion
:::info

When you want to use an endpoint **not** hosted by an MLflow AI Gateway or Databricks, you can create a custom Python function following the [Evaluating with a Custom Function](#evaluating-with-a-custom-function) guide and use it as the ``model`` argument.

:::
```